applepay/utils.py

```diff
@@ -37,10 +57,12 @@ def retrieve_signature_signing_time(signature):
 def signing_time_is_valid(signature, current_time, threshold):
     """ Given a detached top-level CMS signature, validate the 'signingTime'
     attribute against the current time and a time-delta threshold.
-
     If the difference between the current time and the 'signingTime' exceeds
     the threshold, the token should be considered invalid.
 
+    This function is deprecated. Use of `verify_signature` with an optional threshold
+    is preferred.
```

Comment:
Not 100% sure yet, but since this project uses semantic versioning it would be a change to include for the first major release of this lib.